spec.html

```diff
@@ -19327,7 +19333,7 @@ <h1>GetSuperConstructor ( ): an ECMAScript language value</h1>
         <h1>
           MakeSuperPropertyReference (
             _actualThis_: an ECMAScript language value,
-            _propertyKey_: a property key,
+            _propertyValue_: an ECMAScript language value,
```

Comment:
I find `_propertyValue_` is confusing since it's still a key (just not yet ToPropertyKey'd).